openslides_backend/action/actions/meeting/clone.py

```diff
@@ -51,6 +59,19 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         ) and committee_id != self.get_meeting_from_json(meeting_json)["committee_id"]:
             self.get_meeting_from_json(meeting_json)["committee_id"] = committee_id
 
+        # pre update the meeting
+        for field in (
+            "welcome_title",
+            "description",
+            "start_time",
+            "end_time",
+            "location",
+            "organization_tag_ids",
```

Comment:
extract into array so you can deduplicate with the payload above, e.g. `updatable_fields`

Reply:
Extract the fields into "updatable_fields".